src/types/Client.d.ts

```diff
@@ -25,4 +26,5 @@ export default class Client {
   query<T = object>(expr: ExprArg, options?: QueryOptions): Promise<T>
   paginate(expr: Expr, params?: object, options?: QueryOptions): PageHelper
   ping(scope?: string, timeout?: number): Promise<string>
+  stream(expr: Expr, options?: { fields?: string[] }): Subscription
```

Comment:
Not a requirement. More of a question: is it possible to specify the valid values for this array?